src/components/services-page/AreWeAFit.tsx

```diff
@@ -0,0 +1,108 @@
+            focused on finding partnerships that work for both sides of the relationship, where both
+            parties are intensely focused on building remarkable software.
+          </p>
+          <p className="are-we-a-fit__text">
+            If you&#8217;re wanting a relationship with a software team where we work collaboratively and
+            iteratively to build the right software product for the people you serve, you&#8217;re in the
+            right place.
+          </p>
+          <p className="are-we-a-fit__text">
+            We&#8217;re eager to collaborate with clients passionate about deploying impactful software
+            products using a methodical, data-driven approach.
+          </p>
+          <p className="are-we-a-fit__text">
+            Here&#8217;s what makes you an ideal partner for LaunchWare:
+          </p>
+          <ul className="are-we-a-fit__list">
+            <li className="are-we-a-fit__list-item">
+              <FontAwesomeIcon className="are-we-a-fit__check" icon={faCheck} />
+              <span>
+                You embrace Agile principles, valuing adaptability, early delivery, and ongoing
+                improvement.
+              </span>
+            </li>
+            <li className="are-we-a-fit__list-item">
+              <FontAwesomeIcon className="are-we-a-fit__check" icon={faCheck} />
+              <span>
+                You&#8217;re enthusiastic about co-creating a shared vision for your product, rather than
+                handing off a list of requirements.
+              </span>
+            </li>
+            <li className="are-we-a-fit__list-item">
+              <FontAwesomeIcon className="are-we-a-fit__check" icon={faCheck} />
+              <span>
+                You understand the value of transparent communication, be it daily stand-ups or
+                sprint reviews.
+              </span>
+            </li>
+            <li className="are-we-a-fit__list-item">
+              <FontAwesomeIcon className="are-we-a-fit__check" icon={faCheck} />
+              <span>
+                You&#8217;re willing to be actively involved in the development process, from planning to
+                deployment.
+              </span>
+            </li>
+            <li className="are-we-a-fit__list-item">
+              <FontAwesomeIcon className="are-we-a-fit__check" icon={faCheck} />
+              <span>You&#8217;re searching for a long-lasting partnership, not just a one-off gig.</span>
+            </li>
+            <li className="are-we-a-fit__list-item">
+              <FontAwesomeIcon className="are-we-a-fit__check" icon={faCheck} />
+              <span>
+                You appreciate the importance of data-driven decision-making in software
+                development.
+              </span>
+            </li>
+            <li className="are-we-a-fit__list-item">
```

Comment:
FYI we could probably use a before css hook to inject this bullet  - something like this: https://stackoverflow.com/questions/13354578/custom-li-list-style-with-font-awesome-icon nonblocking commentary tho

Reply:
I invested time in this, exploring various integrations/features of plain old css and tailwind, images and font awesome, without success. Given that you said it's non-blocking, I created a story for it to come back to it.

Reply:
good judgment yeah it's not a huge deal. thanks.